smoketest/addrules_test.go

```diff
@@ -71,18 +71,18 @@ func TestAddRules(t *testing.T) {
 	d.ExpectSMS("testing")
 	tw.WaitAndAssert()
 
-	h.FastForward(time.Minute)
+	h.FastForward(30 * time.Minute)
```

Comment:
We do use the same value in most places, but they are actually specific to each test event rather than being a special/generic value that we'd want to put in a constant.

All the generic fast-forward stuff is done in the WaitAndAssert logic, but each one of these is determined by the init SQL for that test, which could change with each test and frequently does when debugging. For example, in the SQL code we may specify 30-minute rules, or 15 and 45, so there is a need to FF 30 minutes. The 30-minute value only has meaning because of what was configured for that test and is important when reading the test to see how far time is advancing.

Basically it's convention driving the same 30 minutes for each, and not something we could change globally without rewriting/updating each individual test. Plus, knowing how much FF each step of the way is super helpful when debugging.